docs/how-to-use-gvfs.md

```diff
@@ -335,17 +382,24 @@ to recompile the native libraries like `libhdfs` and others, and completely repl
+| `oauth2_server_uri`        | The auth server URI for the Gravitino client when using `oauth2` auth type.                                                                              | (none)        | Yes if you use `oauth2` auth type | 0.7.0-incubating |
+| `oauth2_credential`        | The auth credential for the Gravitino client when using `oauth2` auth type.                                                                              | (none)        | Yes if you use `oauth2` auth type | 0.7.0-incubating |
+| `oauth2_path`              | The auth server path for the Gravitino client when using `oauth2` auth type. Please remove the first slash `/` from the path, for example `oauth/token`. | (none)        | Yes if you use `oauth2` auth type | 0.7.0-incubating |
+| `oauth2_scope`             | The auth scope for the Gravitino client when using `oauth2` auth type with the Gravitino Virtual File System.                                            | (none)        | Yes if you use `oauth2` auth type | 0.7.0-incubating |
+| `s3_endpoint`              | The endpoint of the AWS s3.                                                                                                                              | (none)        | Yes if it's a s3 fileset          | 0.7.0-incubating |
+| `s3_access_key_id`         | The access key of the AWS s3.                                                                                                                            | (none)        | Yes if it's a s3 fileset          | 0.7.0-incubating |
+| `s3_secret_access_key`     | The secret key of the AWS s3.                                                                                                                            | (none)        | Yes if it's a s3 fileset          | 0.7.0-incubating |
+| `gcs_service_account_file` | The path of GCS service account JSON file.                                                                                                               | (none)        | Yes if it's a gcs fileset         | 0.7.0-incubating |
+| `oss_endpoint`             | The endpoint of the Aliyun oss.                                                                                                                          | (none)        | Yes if it's a oss fileset         | 0.7.0-incubating |
+| `oss_access_key_id`        | The access key of the Aliyun oss.                                                                                                                        | (none)        | Yes if it's a oss fileset         | 0.7.0-incubating |
+| `oss_secret_access_key`    | The secret key of the Aliyun oss.                                                                                                                        | (none)        | Yes if it's a oss fileset         | 0.7.0-incubating |
```

Comment:
You'd better also split into several tables like Java gvfs. also please give an example here.

Reply:
done.